scenarios3/08_Lost.cfg

```diff
@@ -321,14 +378,35 @@
                 bonus=no
             [/gold_carryover]
         [/objectives]
+
+        [micro_ai]
+            side=4
+            ai_type=return_guardian
+            action=add
+            [filter]
+            [/filter]
+            return_x,return_y=31,67
+        [/micro_ai]
+#ifdef HARD
+        [terrain]  # villages for the dragon's defenders
+            terrain=Uu^Vu
+            x=25,26,29,32,36,37
+            y=68,64,70,70,69,66
+        [/terrain]
+#endif
     [/event]
     {RECALL_HEROES}
+
     [event]
         name=turn 2
+        {PLACE_ITEM 18 87 55}  # Ring of Iced Veins - cold damage, slows and fire resist, just the thing if you want to take on a fire dragon
+        {PLACE_ITEM 97 87 68}  # Boots of Fleeing - I could use those to approach the dragon's castle!
+#ifndef HARD
+        {PLACE_ITEM 259 82 66} # Stolen Potion - skirmisher?  I could use that to approach the dragon's castle!
+        {PLACE_ITEM 527 77 69} # Hey!  Look over here!  Over here, next to the opening to the area with the hidden loot.
+#endif
         {RARE_ITEM 20 53}
-        {RARE_ITEM 87 55}
         {RARE_ITEM 47 47}
-        {RARE_ITEM 87 55}
```

Comment:
The goal was to replace 2 of the super rare items with items that are helpful to complete the scenario but not particularly unusual?

Reply:
Yes.  They're not great items, but they aren't bad. I'd rarely let slow and penetrates sit unused on the inventory list.  The RoIV also sets melee to cold, which negates suck, which keeps me from giving RoIV to E, which is probably a good thing.